### PR TITLE
Update the Chip design of the Identity Server console

### PR DESCRIPTION
**Pull request by Achintha444**
### Purpose
Update the Chip design of the Identity Server console.

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -2077,7 +2077,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     { applicationConfig.advancedConfigurations.showHybridFlowFeatureStatusChip && (
                                         <div className="oxygen-chip-div" >
                                             <Chip
-                                                label={ t("common:new").toUpperCase() }
+                                                label={ t("common:new") }
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/forms/inbound-oidc-form.tsx:2080]: If possible, shall we update this to get the label value from FeatureStatusLabel enum?

File: features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx [hunk 2]
```diff
@@ -17,11 +17,13 @@
  */
 
 import Chip from "@oxygen-ui/react/Chip";
-import { GearIcon } from "@oxygen-ui/react-icons";
+import { DiamondIcon, GearIcon } from "@oxygen-ui/react-icons";
 import { FeatureStatus, FeatureTags, useCheckFeatureStatus, useCheckFeatureTags } from "@wso2is/access-control";
-import { ELK_RISK_BASED_TEMPLATE_NAME } from "@wso2is/admin.authentication-flow-builder.v1/constants/template-constants";
+import { ELK_RISK_BASED_TEMPLATE_NAME } from
+    "@wso2is/admin.authentication-flow-builder.v1/constants/template-constants";
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx:23]:
```suggestion
import {
    ELK_RISK_BASED_TEMPLATE_NAME
} from "@wso2is/admin.authentication-flow-builder.v1/constants/template-constants";
```

**Review by pavinduLakshan — APPROVED** [review]

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (Achintha444) [reply features/admin.applications.v1/components/forms/inbound-oidc-form.tsx:2080]: ack

Comment (Achintha444) [reply features/admin.applications.v1/components/forms/inbound-oidc-form.tsx]: fixed with https://github.com/wso2/identity-apps/pull/6617/commits/d1a5f52eb01ef2f57281cd03d75785ea99528e6b

File: features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx (hunk 2 above)
Comment (Achintha444) [reply features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx]: fixed with https://github.com/wso2/identity-apps/pull/6617/commits/76f2cda14bb048e19d25ad89ad39414d245ed059

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (pavinduLakshan) [reply features/admin.applications.v1/components/forms/inbound-oidc-form.tsx]: Thanks

**Review by pavinduLakshan — DISMISSED** [review]

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**